core/src/main/java/tech/tablesaw/io/DataFrameReader.java

```diff
@@ -123,6 +123,9 @@ public Table html(String url) throws IOException {
         return new HtmlTableReader().read(url);
     }
     
+    // The xlsx methods use the optional dependency org.apache.poi:poi-ooxml.
+    // A modules that use them must itself add this dependency.
```

Comment:
"A modules that use them must itself add this dependency" sounds wrong

Maybe combine the two sentences into "Modules that call this method must add the optional dependency org.apache.poi:poi-ooxml"

Also, I'd suggest making it a Javadoc comment so that it shows up in the Javadoc and add it to both methods:
```
/**
 * Modules that call this method must add the optional dependency org.apache.poi:poi-ooxml
 */ 
```